src/AutoRest.CSharp/Mgmt/Models/MgmtRestOperation.cs

```diff
@@ -93,9 +99,15 @@ internal record MgmtRestOperation
 
         public bool IsLongRunningOperation => _isLongRunning.HasValue ? _isLongRunning.Value : Operation.IsLongRunning;
 
+        public bool IsFakeLro => IsLongRunningOperation && !Operation.IsLongRunning;
```

Comment:
I personally prefer `IsFakeLongRunningOperation` for this property comparing to the property of `IsLongRunningOperation`. What do you think? Or we could rename the other to `IsLro`